pom.xml

```diff
@@ -136,18 +136,6 @@
             <version>2.0</version>
             <type>jar</type>
         </dependency>
-        <dependency>
-            <groupId>com.google.guava</groupId>
-            <artifactId>guava</artifactId>
-            <version>19.0</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.jenkins-ci.plugins</groupId>
-            <artifactId>matrix-project</artifactId>

```

Comment:
I hope this one was included on puprose, but don't remember why

Reply:
Yea, I've found this conversation, pointing to this bug

https://github.com/jenkinsci/stashnotifier-plugin/pull/201/files#r271855463

https://issues.jenkins-ci.org/browse/JENKINS-27826

It's not relevant for now?

Reply:
I will have a look again.

Reply:
Yes it is fine now without explicitly declaring this dependency. Otherwise tests would fail (I double checked the mentioned pull request).

The "git" plugin has "matrix-project" as dependency since v2.3 - see https://github.com/jenkinsci/git-plugin/commit/c8e7f22488c7370a5727d6d684048ff6e78429c2#diff-600376dffeb79835ede4a0b285078036R262-R266.

The "git" dependency was upgraded via https://github.com/jenkinsci/stashnotifier-plugin/commit/976b392c2dd29cb08867d3befe8ea8081ada17cb#diff-600376dffeb79835ede4a0b285078036R93.